packaging/rpm/packages/licences.spec

```
@@ -0,0 +1,8 @@
+# Licences install commands
+%global licences_install_commands \
```

RuboCop (1.81.1)

[fatal] unterminated string meets end of file
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)